.github/workflows/tests.yml

```
@@ -0,0 +1,36 @@
+name: Tests
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+concurrency:
+  group: ${{ github.repository }}-${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+jobs:
+  tests_linux:
+    runs-on: ${{ matrix.os }}
+    needs: []
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9, "3.10"]
+        os: ["ubuntu-latest", "macOS-latest", "windows-2019"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Entanglement Forging
+        run: |
+            set -e
+            pip install -r requirements.txt
+            pip install -r requirements-dev.txt
+            pip freeze
+        shell: bash
+      - name: Run Tests
+        run: |
+          set -e
+          treon docs/ --threads 5
+        shell: bash
```

Same as above comment, about `set -e` and `shell: bash`

```suggestion
          treon docs/ --threads 5
```